syntax = "proto3";
package GenomEvaluation;

service GenomEvaluation {
  rpc GetIndividual(Genom) returns (Individual) {};
  rpc GetIndividualMock(Genom) returns (Individual) {};
}

message Genom {
  repeated float gene = 1 [packed=true];
}

message Individual {
  Genom genom = 1;
  float evaluation = 2;
}

message Generation {
  repeated Individual individuals = 1;
}